bin/setup.js

```diff
@@ -15,7 +15,15 @@ if (saveRequested) {
   var packageJson = require(path.join(pathToAppRoot, 'package.json'))
 
   packageJson.scripts = packageJson.scripts || {}
-  packageJson.scripts[ 'start' ] = 'hoodie'
+
+  if (packageJson.scripts[ 'start' ]) {
+    log.info('setup', 'start script already set to "' + packageJson.scripts[ 'start' ] +
+    ', you can start hoodie with "npm run start-hoodie" instead')
+
+    packageJson.scripts[ 'start-hoodie' ] = packageJson.script[ 'start' ]
```

Comment:
Correct me if I am wrong, but:

1. Sometimes you say `packageJson.scripts`, other times you say `packageJson.script`. I think it should always be `packageJson.scripts`?
2. Why are we setting `start-hoodie` to `start`. That feels like the opposite of what we want to do. Shouldn't we be setting `start-hoodie` to `hoodie`?
3. I think there might be an opportunity to write a test for this, somewhere in our unit tests, to prove that this works?

Wondering if @gr2m has any thoughts on my comments above.
Thanks so much for starting this PR ⭐️